support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -150,6 +163,24 @@ export function SupporterPlusLandingPage({
 	};
 	const heading = <LandingPageHeading />;
 
+	const participations = initAbTests(
```

Comment:
You shouldn't need to call `initAbTests` here, the user's AB test participations have already been allocated and added to the Redux state. You should be able to use the `isUserInAbVariant` function from 'helpers/redux/commonState/selectors' to determine if a user in a test cohort rather than looking up `participations`. EG.

Replace...

```
const [supportOnceDisplay, setSupportOnceDisplay] = useState(
		participations.singleLessProminent === 'variant',
	);
```

With...

```
const [supportOnceDisplay, setSupportOnceDisplay] = useState(
		isUserInAbVariant('singleLessProminent', 'variant')
	);
```

Reply:
Ok, quicker from redux store than participations

Reply:
changes made